include/linux/usb.h

```diff
@@ -1612,14 +1612,18 @@ struct urb {
 #include <soc/starfive/vic7100.h>
 static inline void cdns_flush_dcache(unsigned long start, unsigned long len)
 {
+	if (start == 0UL)
+		return;
+
```

Comment:
I thought about it.  I will change this to only work on the virtual `NULL` values.  That is where we see the issue.